spanner_orm/tests/field_test.py

```diff
@@ -146,10 +160,13 @@ def test_string_array_is_deprecated_and_equivalent_to_array_of_string(self):
       'INT64',
       'FLOAT64',
       'STRING(MAX)',
+      'STRING(10)',
       'TIMESTAMP',
       'BYTES(MAX)',
+      'BYTES(10)',
       'ARRAY<INT64>',
       'ARRAY<STRING(MAX)>',
+      'ARRAY<STRING(10)>',
   )
   def test_ddl_to_field_type_to_ddl(self, ddl: str):
     self.assertEqual(field.field_type_from_ddl(ddl).ddl(), ddl)
```

Comment:
(nit) Please make test_field_type_from_ddl_invalid parameterized and add a case or two for invalid lengths.

Reply:
Done.